apis/Google.Cloud.Firestore.Admin.V1/Google.Cloud.Firestore.Admin.V1.Snippets/FirestoreAdminClientSnippets.g.cs

```diff
@@ -37,25 +37,25 @@ public void CreateIndexRequestObject()
             CreateIndexRequest request = new CreateIndexRequest
             {
                 ParentAsCollectionGroupName = CollectionGroupName.FromProjectDatabaseCollection("[PROJECT]", "[DATABASE]", "[COLLECTION]"),
-                Index = new Index(),
+                Index = new gcfav::Index(),
```

Comment:
I don't understand why this has changed. All that's changed in the generator is using .NET Core 3.1 instead of 2.2.

Reply:
I suspect it's because of the new `System.Index` struct.
In which case, this is working as expected :)

Reply:
Aha, makes sense. Will merge, thanks.